assets/dev/scss/frontend/_forms.scss

```diff
@@ -8,7 +8,7 @@
 
 	&.elementor-labels-above {
 
-		> .elementor-field-group {
+		.elementor-field-group {
```

Comment:
Insert a temporary selector in a patch version of pro + removal of it in version 3.0